bin/app.rb

```diff
@@ -2,7 +2,8 @@
 class App
   include AwsLoader
 
-  OPERATIONS = ['create', 'delete', 'list']
+  OPERATIONS = ['create', 'delete', 'list', 'copy']
+  BUCKET_URI_REGEX = /^(s3:\/\/)/
```

Comment:
The regex seems rather skinny. Have a larger matching conditions. Regions I believe are in the url.

Reply:
yes.. but not the bucket location.. we don't need the region in that case. typically s3 URI's are of the format `s3://<bucket-name>/path-to-s3-object`. In this case the region does not come into picture.